code/_helpers/matrices.dm

```diff
@@ -25,6 +25,12 @@
 		//doesn't have an object argument because this is "Stacking" with the animate call above
 		//3 billion% intentional
 
+/atom/proc/shake_animation(var/intensity = 8)
+	var/init_px = pixel_x
+	var/shake_dir = pick(-1, 1)
+	animate(src, transform=turn(matrix(), intensity*shake_dir), pixel_x=init_px + 2*shake_dir, time=1)
+	animate(transform=null, pixel_x=init_px, time=6, easing=ELASTIC_EASING)
```

Comment:
this doesn't have the return value the proc it's replacing had. iirc something else in the code relied on it